rest/route/github.go

```diff
@@ -754,7 +754,7 @@ func (gh *githubHookApi) createVersionForTag(ctx context.Context, pRef model.Pro
 	}
 
 	if !pRef.AuthorizedForGitTag(ctx, tag.Pusher, token, pRef.Owner, pRef.Repo) {
-		grip.Debug(message.Fields{
+		grip.Error(message.Fields{
```

Comment:
I'm actually against making this an error (or returning an error), because it _doesn't_ point to an error in Evergreen behavior, which we should keep these limited to. People often use tags for things that aren't Evergreen.

Reply:
Gotcha. I knew this would error but thought that returning an error from the webhook response wouldn't be that disruptive or even visible to the user, but it does make sense that this isn't the kind of thing we should error out on.

Reply:
This grip statement still needs to be changed (we don't want this to count towards our error count since it's not actually an error)

Reply:
Changed back